unit_tests/testAndersonAcceleration/testAndersonAcceleration.f90

```diff
@@ -20,16 +20,15 @@ PROGRAM testAndersonAcceleration
 
 TYPE(ExceptionHandlerType),TARGET :: e
 TYPE(MPI_EnvType) :: mpiTestEnv
-TYPE(ParamType) :: pList, optList
+TYPE(ParamType) :: pList,optList
 TYPE(AndersonAccelerationType) :: testAndAcc
 
-
 !> set up default parameter list
 CALL optList%clear()
-CALL optList%add('AndersonAccelerationType->n',5_SIK)
```

Comment:
so it looks like you deleted an example where I had worked out the solution for multiple iterations but didn't really replace it with anything.